java.lang.NoClassDefFoundError: cats/FunctorFilter

I am creating a project from https://github.com/frees-io/freestyle-template.g8 and adding the follow plugins, I am getting the following error.

```
resolvers += Resolver.sonatypeRepo("releases")
resolvers += Resolver.bintrayRepo("beyondthelines", "maven")

addSbtPlugin("io.frees" % "sbt-freestyle" % "0.3.6")
addSbtPlugin("io.frees" % "sbt-frees-protogen" % "0.0.10")
```

```
info] Loading settings from build.sbt,version.sbt ...
[error] java.lang.NoClassDefFoundError: cats/FunctorFilter
[error]         at sbtorgpolicies.settings.DefaultSettings.$anonfun$orgCommonDefaultSettings$4(DefaultSettings.scala:74)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error]         at sbt.internal.util.EvaluateSettings$MixedNode.evaluate0(INode.scala:214)
[error]         at sbt.internal.util.EvaluateSettings$INode.evaluate(INode.scala:159)
[error]         at sbt.internal.util.EvaluateSettings.$anonfun$submitEvaluate$1(INode.scala:82)
[error]         at sbt.internal.util.EvaluateSettings.sbt$internal$util$EvaluateSettings$$run0(INode.scala:93)
[error]         at sbt.internal.util.EvaluateSettings$$anon$3.run(INode.scala:89)
[error]         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error]         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error]         at java.lang.Thread.run(Thread.java:748)
[error] Caused by: java.lang.ClassNotFoundException: cats.FunctorFilter
[error]         at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
[error]         at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[error]         at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[error]         at sbtorgpolicies.settings.DefaultSettings.$anonfun$orgCommonDefaultSettings$4(DefaultSettings.scala:74)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error]         at sbt.internal.util.EvaluateSettings$MixedNode.evaluate0(INode.scala:214)
[error]         at sbt.internal.util.EvaluateSettings$INode.evaluate(INode.scala:159)
[error]         at sbt.internal.util.EvaluateSettings.$anonfun$submitEvaluate$1(INode.scala:82)
[error]         at sbt.internal.util.EvaluateSettings.sbt$internal$util$EvaluateSettings$$run0(INode.scala:93)
[error]         at sbt.internal.util.EvaluateSettings$$anon$3.run(INode.scala:89)
[error]         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error]         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error]         at java.lang.Thread.run(Thread.java:748)
[error] java.lang.NoClassDefFoundError: cats/FunctorFilter
```

In the release 0.0.8 works fine but after generate the release 0.0.9 I am getting this error.

This is the PR where probably we can start to debug something https://github.com/frees-io/sbt-freestyle-protogen/pull/5/files

--- a/version.sbt
+++ b/version.sbt
@@ -1,1 +1,1 @@
-version in ThisBuild := "0.0.13-SNAPSHOT"
+version in ThisBuild := "0.0.13"

